### DISCUSSION TITLE
Clientside sub only mode and animated emotes

### DISCUSSION THREAD

**Discussion by rtyuii** [Chatterino/chatterino2#2913] [Ideas]
/localsub needsss to be a thing for chatters if possible and then theres the new animated twitch emotes but i think they're working on it :)

**Comment by KararTY** [comment]
> [...] and then theres the new animated twitch emotes but i think they're working on it :)

New nightly displays animated twitch emotes. Get it over here: https://github.com/Chatterino/chatterino2/releases

**Comment by zneix** [comment]
Your first request is a duplicate of issue #1011, please use filters for that.
Your second request is already implemented. As KararTY said, you should download the newest [nightly build](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build) which supports them.

By the way, next time please open separate issues / discussions for each issue / suggestion.

P.S. updated your discussion title to be most descriptive.
